doc/release-notes/0.9/0.9.md

```diff
@@ -0,0 +1,184 @@
+<td valign="top">Extra page of memory no longer required when specifying an object heap size that is a multiple of the page size.</td>
+<td valign="top">Linux on Z and z/OS</td>
+<td valign="top">This change avoids wasting memory or calculating a heap size that is 16 bytes less than the page size.</td>
+</tr>-->
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1668">#1668</a></td>
+<td valign="top">Idle tuning feature enabled.</td>
+<td valign="top">Linux on Z, Linux on Power systems</td>
+<td valign="top">Ability to maintain a minimal memory footprint during the life time of an application.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1517">#1517</a></td>
+<td valign="top">Changes to the default Java heap size for applications that run inside a container.</td>
+<td valign="top">Linux</td>
+<td valign="top">By default, 75% of the physical memory is allocated for the maximum Java heap setting when the <code>-XX:+UseContainerSupport</code> option is set.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1500">#1500</a></td>
+<td valign="top">Support for nested jar files</td>
+<td valign="top">All platforms</td>
+<td valign="top">Improvements to the `com.ibm.oti.shared` API for manipulating nested jar files, such as used by Spring.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1786">#1786</a></td>
+<td valign="top">Changes to OpenJ9 <code>java.lang.String</code> class and use of OpenJDK <code>StringBuffer/StringBuilder</code></td>
+<td valign="top">OpenJDK 9+, all platforms</td>
+<td valign="top">To match the behavior of OpenJDK, <code>java.lang.String</code> no longer has a count field, which changes the way that <code>String.subString()</code> works compared to Java 8. Similarly <code>StringBuffer</code and <code>StringBuilder</code> do not share the value array with any <code>String</code> created by <code>toString()</code>.</td>
+</table>
+
+
+## Known Issues
+
+The v0.9.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/378">#378</a></td>
+<td valign="top">DDR support</td>
+<td valign="top">AIX</td>
+<td valign="top">Inability to diagnose problems with the VM, garbage collector, or JIT.</td>
+<td valign="top">None</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/54">#54</a></td>
+<td valign="top">Signal handling</td>
+<td valign="top">All platforms</td>
+<td valign="top">Some signals not being handled correctly by the VM.</td>
```

Comment:
It should be in the notable changes that the 0.9.0 release adds sun.misc.Signal / jdk.internal.misc.Signal support for SIGCONT, SIGHUP, SIGINT, SIGTERM on Linux and AIX. On Windows SIGINT runs the shutdown handlers, but cannot be configured via sun.misc.Signal / jdk.internal.misc.Signal. 
@babsingh please confirm I got this correct.

Reply:
`On Windows SIGINT` -> `On Windows SIGINT, SIGTERM and Ctrl+C`
rest looks correct. i added more details below.